django_autoconfig/app_settings.py

```diff
@@ -5,6 +5,9 @@
 #: Extra URLs for autourlconf
 AUTOCONFIG_EXTRA_URLS = getattr(settings, 'AUTOCONFIG_EXTRA_URLS', ())
 
+#: A list/tuple of apps to exclude from the autourlconf
```

Comment:
Will this be surfaced in the docs? I don't think we're currently displaying the autogenerated page from this file